website/.env

```diff
@@ -12,3 +12,6 @@ NEXTAUTH_SECRET=O/M2uIbGj+lDD2oyNa8ax4jEOJqCPJzO53UbWShmq98=
 EMAIL_SERVER_HOST=localhost
 EMAIL_SERVER_PORT=1025
 EMAIL_FROM=info@example.com
+
+DISCORD_CLIENT_ID=1058355952459452446
```

Comment:
We shouldn't check these in.  Can you remove them.

Reply:
If you want to keep them locally, you can put this in `.env.local`

Reply:
Oh ok, I will remove and push the changes

Reply:
Oh, but I have made some commits after that one too. Will those be affected too? @bitplane

Reply:
@KarthikRaju391 
No, it'll just push the rewritten history of your branch. It might mean that some review comments can't link to code that's changed anymore, I'm not sure. There's 3 repos going on:
1. your local repo and the branch you're working, which has a history containing each commit
2. Your fork on github has the same thing
3. This repo's main has a history of all commits

Here's how they're applied:

* `git rebase -i` will rewrite the history of `1` 
* `git push -f` will overwrite `3` with the contents of `2`
* This pull request is a GitHub thing that requests we pull code from `2` into `3`, it's based on the name of `2` not its history

We could do an alternative method: When merging your changes into `main`, we choose the "squash commits when merging" approach, which will remove all your commits and replace them with a single squashed commit. But that kinda sucks because your commits will be gone, so you as far as GitHub is concerned you won't be credited as one of the project's authors. So the rebase + force push is the best option